demo/src/pages/index.astro

```diff
@@ -1,22 +1,23 @@
 ---
-import Document from '../layouts/Document.astro'
+import Document from "../layouts/Document.astro";
 ---
+
 <Document title="Astro Flow">
 	<h1>Astro Flow</h1>
 	<p>
 		<strong>Astro Flow</strong> lets you use control flow statements in Astro.
 	</p>
 	<hr />
 	<section>
-		<h2>
-			Demonstrations
-		</h2>
+		<h2>Demonstrations</h2>
 		<ul>
 			<li>
 				<a href="./for/">The <code>&lt;For&gt;</code> component</a>
 			</li>
 			<li>
-				<a href="./switch/">The <code>&lt;Switch&gt;</code> component.</a>
+				<a href="./switch/"
+					>The <code>&lt;Switch&gt;</code> component.</a
+				>
```

Comment:
This doesn't seem related to any of the changes you want to make. Out of curiosity, has a code formatter done this, or is this change something you believe improves the readability of the document?

Reply:
I think it's due to the formatter. I'm going to change this